dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CustomizedExceptionText.java

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.webcontainer.servlet31.fat.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.meterware.httpunit.WebConversation;
+import com.meterware.httpunit.WebResponse;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class CustomizedExceptionText {
+
+    private static final Logger LOGGER = Logger.getLogger(CustomizedExceptionText.class.getName());
+    private static final String CUSTOM_ERROR_APP_NAME = "CustomErrorTestApp";
+
+    @Server("servlet31_customizedExceptionText")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        WebArchive customErrorApp = ShrinkHelper.buildDefaultApp(
+            CUSTOM_ERROR_APP_NAME + ".war",
+            "com.ibm.ws.webcontainer.servlet_31_fat.customerrortest.war.test.servlets"
+        );
+        customErrorApp = (WebArchive) ShrinkHelper.addDirectory(
+            customErrorApp,
+            "test-applications/" + CUSTOM_ERROR_APP_NAME + ".war/resources"
+        );
+        ShrinkHelper.exportDropinAppToServer(server, customErrorApp);
+    }
+
+    @AfterClass
+    public static void cleanup() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer();
+        }
+    }
+    /**
+     * Verify that the customized exception text configured in server.xml
+     * is returned instead of the default
+     * "SRVE0218E: Forbidden" message
+     * when the application sends a 403 Forbidden error.
+     */
+    @Test
+    public void tes403CustomMessage() throws Throwable {
+        server.stopServer();
+        server.setServerConfigurationFile("customizedExceptionTest/server-test-403-custom-message.xml");
+        server.startServer();
+        server.waitForStringInLogUsingMark("CWWKT0016I.*" + CUSTOM_ERROR_APP_NAME + ".*");
```

Comment:
Do we really need to start and stop the server? Can't we just update the service configuration and wait for the config updates to finish like we do in other tests?